lib/main.js

```diff
@@ -2,38 +2,43 @@
 Copyright (C) 2018 The Trustees of Indiana University
 SPDX-License-Identifier: BSD-3-Clause
 */
-import * as Alert from '../src/components/Alert/index';
+import * as RRAlert from '../src/components/Alert/index';
```

Comment:
I suspect you don't need to do this `import` then `export` step. You may be able to do it in one line. Have you tried this? This approach would make this file easier to maintain.

```js
export * from '../src/components/Alert/index';
```

Reply:
This appears to work.  It does have the drawback is a loss of purpose control on exports (components will be automatically instead of having a speedbump here).  Can even shorten it to `export * from '../src/components/index';` but would lose even more purpose control.